src/foam/nanos/auth/User.js

```diff
@@ -203,9 +203,11 @@ foam.CLASS({
     },
     {
       class: 'Password',
-      name: 'password',
+      name: 'desiredPassword',
+      label: 'Password',
       displayWidth: 30,
       width: 100,
+      storageTransient: true,
       validateObj: function (password) {
         var re = /^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9]).{7,32}$/;
```

Comment:
Not a blocking comment. Curious if during password entry (or after failure) we tell them what the requirements are.

Reply:
Not sure if we do or not